### PR TITLE
Update design and copy

### PR DESCRIPTION
**Pull request by vickymg**

### CODE REVIEW

File: mtp_send_money/templates/send_money/send-money.html [hunk 1]
```diff
@@ -18,7 +21,8 @@ <h1 class="heading-xlarge">{% trans 'How much money do you want to send?' %}</h1
     {% with field=form.amount %}
       <div class="mtp-amount">
         <fieldset>
-          <legend class="form-label-bold">{% trans "How much are you sending?" %}</legend>
+          <legend class="form-label">{% trans 'Amount' %}</legend>
+          <span class="mtp-label-secondary">{% trans 'You’ll receive a refund if prisoner details are incorrect' %}</span>
```
Comment (ushkarev) [inline mtp_send_money/templates/send_money/send-money.html]: i think `form-hint` works closely enough

File: mtp_send_money/assets-src/stylesheets/app.scss [hunk 2]
```diff
@@ -46,8 +53,40 @@ form .form-group .form-hint {
...
+  padding-left: 30px;
+}
+
 // Application-specific
 
+.mtp-instructions {
+  width: 65%;
+}
+
+.mtp-label-secondary {
```
Comment (ushkarev) [inline mtp_send_money/assets-src/stylesheets/app.scss]: not needed now i think

File: mtp_send_money/templates/send_money/bank-transfer.html [hunk 3]
```diff
@@ -1,51 +1,43 @@
...
+    <h1 class="heading-large">{% trans 'Prisoner bank transfer reference' %}</h1>
+    <p>{% trans 'Keep this for future transfers to this prisoner' %}</p>
+    <strong>{{ bank_transfer_reference }}</strong>
+  </div>
 
-  <section>
-    <h2 class="heading-large">{% trans '1. Log in to your online banking or mobile banking app, or telephone your bank' %}</h2>
-  </section>
+  <div class="mtp-print">
+    <img src="{% static 'images/icon-file-download.png' %}"
```
Comment (ushkarev) [inline mtp_send_money/templates/send_money/bank-transfer.html]: i *think* in the case of icons, the typical thing would be to add it as a background to the `a` element.

something like:
```css
.mtp-print-reference {
background: url(???/icon-file-download.png) no-repeat;
padding-left: ??px;
/* something to force the height to be at least whatever */
}
@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min-device-pixel-ratio: 2) {
background-image: url(???/icon-file-download-2x.png);
background-size: ??px ??px;
}
```

do the GDS style already include some usage of this icon we can use?

File: mtp_send_money/assets-src/stylesheets/app.scss [hunk 4]
```diff
@@ -32,6 +31,14 @@ form .form-group .form-hint {
   margin-top: 0;
 }
 
+img {
```
Comment (ushkarev) [inline mtp_send_money/assets-src/stylesheets/app.scss]: no longer needed if print icon becomes a background image i think. also, this selector is a little too broad anyway :)

File: mtp_send_money/templates/send_money/send-money.html (hunk 1 above)
Comment (vickymg) [inline mtp_send_money/templates/send_money/send-money.html]: sorted this one :-)

**Pull request merged**